pages/rops/components/cancel-link.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default function CancelLink() {
+  if (typeof window !== 'undefined') {
+    if (window.history.length <= 1) {
+      return null;
+    }
+  }
```

Comment:
We probably want to change this section to `if (!window || window.history.length <= 1) {` so that it also returns null when server rendering.

Reply:
if (!window) would throw an error on server as undefined, you need to use typeof window !== undefined (see line above)

Reply:
In which case `if (typeof window === 'undefined' || window.history.length <= 1) {`

The key being that we don't fall through to render a potentially broken `<a>` when rendering on the server.

Reply:
yeah currently its rendered on the server then killed by the client. I'll update it to return null from server

Reply:
Fanks.